### DISCUSSION TITLE
DietPi-Software | RaspAP-webgui

### DISCUSSION THREAD

**Discussion by cocoflan** [MichaIng/DietPi#5425] [Software Request]
https://github.com/billz/raspap-webgui A simple, responsive web interface to control wifi and hostapd on the Raspberry Pi.

RaspAP lets you quickly get a WiFi access point up and running to share the connectivity of many popular Debian-based devices, including the Raspberry Pi. Our popular Quick installer creates a known-good default configuration that “just works” on all current Raspberry Pis with onboard wireless. A responsive interface gives you control over the relevant services and networking options. Advanced DHCP settings, OpenVPN client support, SSL, security audits, themes and multilingual options are included.

RaspAP has been featured on sites such as Instructables, Adafruit, Raspberry Pi Weekly and Awesome Raspberry Pi and implemented in countless projects.

We hope you enjoy using RaspAP as much as we do creating it. Tell us how you use this with your own projects.

**Comment by MichaIng** [comment]
There is an issue open for this already: #2813
Some testing has been done there, let's not split this. I can convert that issue into a discussion as well.

**Reply by cocoflan** [reply]
OK good!
